src/Symfony/Component/VarExporter/Tests/LazyProxyTraitTest.php

```diff
@@ -295,6 +296,31 @@ public function testNormalization()
         $this->assertSame(['property' => 'property', 'method' => 'method'], $output);
     }
 
+    public function testReinitRegularLazyProxy()
+    {
+        $object = $this->createLazyProxy(RegularClass::class, fn () => new RegularClass(123));
+
+        $this->assertSame(123, $object->foo);
+
+        $object::createLazyProxy(fn () => new RegularClass(234), $object);
+
+        $this->assertSame(234, $object->foo);
+    }
+
+    /**
+     * @requires PHP 8.3
```

Comment:
Why 8.3 ? Readonly classes are a PHP 8.2 feature. Is there an incompatibility with 8.2 ?

Reply:
Yes: ProxyHelper refuses to generate lazy proxies before 8.3 because cloning is broken before, see https://wiki.php.net/rfc/readonly_amendments